packages/config/config/devices/0x0258/nas-wr01z.json

```
@@ -0,0 +1,200 @@
+// Shenzhen Neo Electronics Co., Ltd. NAS-WR01Z
+// Wall Plug Switch
+{
+	"manufacturer": "Shenzhen Neo Electronics Co., Ltd.",
+	"manufacturerId": "0x0258",
+	"label": "NAS-WR01Z",
+	"description": "Wall Plug Switch",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x0087"
+		},
+		{
+			"productType": "0x0003",
+			"productId": "0x1087"
+		},
+		{
+			"productType": "0x0003",
+			"productId": "0x2087"
+		},
+		{
+			"productType": "0x0003",
+			"productId": "0x1483"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Meter Report Enable",
+			"description": "This parameter defines Disable/Enable meter report measure data to controller.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable Report",
+					"value": 0
+				},
+				{
+					"label": "Enable Report",
+					"value": 1
+				}
+			]
+		},
+		"2": {
+			"label": "Meter Report Interval",
+			"description": "Interval time reporting data to controller",
+			"unit": "Seconds",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 65535,
+			"defaultValue": 300,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Maxium over-load current",
+			"description": "maximum current the plug can provide to load",
+			"unit": "Ampere",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 16,
+			"defaultValue": 13,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"4": {
+			"label": "Maximum alarm current",
```

Maybe something like
```suggestion
			"label": "Alarm current threshold",
```